testutils/api/useradm.py

```diff
@@ -19,5 +19,6 @@
 URL_LOGIN = '/auth/login'
 URL_SETTINGS = '/settings'
 URL_2FAQR = '/2faqr'
+URL_VERIFY = '/2faverify'
```

Comment:
To follow current convention, I think this one should be `URL_2FAVERIFY`

Reply:
changed.